pkg/subctl/cmd/validate_cni.go

```diff
@@ -87,7 +87,7 @@ func validateCNIInCluster(config *rest.Config, clusterName string, submariner *v
 			" Supported network plugins: %v\n", submariner.Status.NetworkPlugin, supportedNetworkPlugins)
 		status.QueueFailureMessage(message)
 		status.End(cli.Failure)
-		return
+		exitWithErrorMsg(message)
```

Comment:
This will exit when running validate all, right?

Shouldn't we do this in L70. ?

And only after we have checked all the clusters, not only the first one.

Reply:
I agree with @mangelajo

Reply:
+1.  We can have each validate function return a success/fail bool that is checked at the top-level.